torch/_dynamo/variables/torch.py

```diff
@@ -128,9 +128,14 @@ class TorchVariable(VariableTracker):
 
     def __init__(self, value, **kwargs):
         super().__init__(**kwargs)
+        try:
+            if value in tensor_dunder_fns_remap:
+                value = tensor_dunder_fns_remap[value]
+        except TypeError as e:
+            # is it cleaner to somehow exclude unhashable types from the `in` check above?
+            if "unhashable type" not in e.args[0]:
```

Comment:
for my learning: is the bytecode directly generate "unhashable type" for process group so that we need to handle it here when we calling sth like `torch.distributed._functional_collective.all_gather_tnesor`?

Reply:
no, the issue here is that someone wrote code that assumes 'value' will be a function and can be looked up in a dict of functions to remap.  But value isn't always a function and so i had to either check if its a function or check if its hashable to be able to look it up safely.